src/window/window_wrapper/mod.rs

```diff
@@ -104,10 +104,15 @@ impl GlutinWindowWrapper {
         self.windowed_context.window().set_title(&self.title);
     }
 
-    pub fn handle_quit(&mut self) {
-        self.ui_command_sender
-            .send(UiCommand::Quit)
-            .expect("Could not send quit command to bridge");
+    pub fn handle_quit(&mut self, running: &Arc<AtomicBool>) {
+        let prefix = "--remote-tcp";
+        if let None = std::env::args().find(|arg| arg.starts_with(prefix)) {
```

Comment:
Could you change this to access the setting using the new infrastructure that just got merged? An example can be found here: https://github.com/Kethku/neovide/blob/opengl/src/bridge/mod.rs#L144

Reply:
kk, it's done.